### ISSUE TITLE
Wrong compare operator:

### ISSUE THREAD

**Issue by missytake** [deltachat-bot/discourse-login-bot#3]
In https://github.com/deltachat-bot/discourse-login-bot/blob/master/src/web_app.js#L35, the `!==` operator is used to compare the redirect_uri from the URL vs. the config. Unfortunately, this comparison always fails with the following log message:

```
[3/17/2020, 11:17:44 AM] Unknown redirect_uri, denying access: https://support.delta.chat/auth/oauth2_basic/callback
```

Note that I modified the log output to print the `redirect_uri`. Interestingly, the log output shows the same `redirect_uri` both from the config and from discourse.

A [quick research](https://www.w3schools.com/js/js_comparisons.asp) shows: the `!==` operator only returns true if also the type of the compared parameters is the same. But a modified log output shows:

```
[3/17/2020, 11:25:24 AM] Type of params.redirect_uri: string
[3/17/2020, 11:25:24 AM] Type of config.oauth.redirect_uri: object
```

So we either have to compare the two parameters with `!=` instead (which works in production, btw), or make the config value a string instead.

**Comment by missytake** [comment]
Interestingly, in https://github.com/deltachat-bot/discourse-login-bot/blob/master/src/web_app.js#L29, both values are a string, and the `!==` comparison works as expected. So the problem is that `oauth.redirect_uri` in the config is an array, and can contain multiple strings. It should be enough to check whether params.redirect_uri is in the array.

**Comment by pabzm** [comment]
`redirect_uri` should be a string in the config, not an array. Then we don't need a code-fix either, right?

The README still shows an array, that's wrong (fixed in #6).

**Comment by pabzm** [comment]
Closed via  #6
